diff --git a/pyproject.toml b/pyproject.toml
index 2d2436d8..9018c851 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@ dependencies = [
   "update_checker >=0.18",
   "websocket-client >=0.54.0"
 ]
-dynamic = ["version", "description"]
+dynamic = ["description"]
 keywords = ["reddit", "api", "wrapper"]
 license = {file = "LICENSE.txt"}
 maintainers = [
@@ -36,6 +36,7 @@ maintainers = [
 name = "praw"
 readme = "README.rst"
 requires-python = "~=3.8"
+version= "7.8.1"
 
 [project.optional-dependencies]
 ci = ["coveralls"]
